newsfragments/2562.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix `is_dynamic_fee_transaction` and `TRANSACTION_DEFAULTS` when `gas_price_strategy` returns zero
```

Comment:
Hey @VladLupashevskyi... this file should be named `2570.bugfix.rst` as it should relate to the PR number for the `v5` branch.